conf/tidb.yml

```diff
@@ -106,39 +61,10 @@ performance:
   # StmtCountLimit limits the max count of statement inside a transaction.
   # stmt-count-limit: 5000
 
-  # Set keep alive option for tcp connection.
-  # tcp-keep-alive: true
-
-  # Whether support cartesian product.
-  # cross-join: true
-
   # Stats lease duration, which influences the time of analyze and stats load.
   # stats-lease: "3s"
 
-  # Run auto analyze worker on this tidb-server.
-  # run-auto-analyze: true
-
-  # Probability to use the query feedback to update stats, 0 or 1 for always false/true.
-  # feedback-probability: 0.05
-
-  # The max number of query feedback that cache in memory.
-  # query-feedback-limit: 1024
-
-  # Pseudo stats will be used if the ratio between the modify count and
-  # row count in statistics of a table is greater than it.
-  # pseudo-estimate-ratio: 0.8
```

Comment:
It's better to keep them in the config files:
```
run-auto-analyze
feedback-probability
query-feedback-limit
pseudo-estimate-ratio
```

Reply:
https://docs.google.com/spreadsheets/d/1fmY3MSklHb9jYQOv3Djcrhj3Uz97fI06soaMy4OKcRs/edit#gid=0

PM think they should not be visible to user